### PR TITLE
Assets models

### LINKED ISSUE #449: Artwork - Gates
Create proper model for gates.

This is a subtask of #448 - Artwork

### LINKED ISSUE #450: Artwork - Void platform
Create proper model or texture for void platforms.

This is a subtask of #448 - Artwork

### LINKED ISSUE #454: Artwork - Skymap
Add a (rock texture) skymap, so that falling in a pitfall actually looks like you are falling in something else than the void.

This is a subtask of #448 - Artwork

### PR DESCRIPTION
**Pull request by WytzeElhorst**
Adds a model to gates and a texture to voidplatforms.

This closes #449, closes #450, closes #454.

### CODE REVIEW

**Comment by Geweldig** [comment]
Can be merged.

File: src/main/java/nl/tudelft/contextproject/model/entities/VoidPlatform.java [hunk 1]
```diff
@@ -28,6 +30,9 @@ public Spatial getSpatial() {
 		material.setColor("Specular", color);
 		material.setFloat("Shininess", 114f);
 		material.setColor("Ambient", color);
+		material.getAdditionalRenderState().setBlendMode(BlendMode.Alpha);
+	    this.spatial.setQueueBucket(Bucket.Transparent);
```
Comment (ChielBruin) [inline src/main/java/nl/tudelft/contextproject/model/entities/VoidPlatform.java]: Missed a tab

**Comment by ChielBruin** [comment]
If the missing tab is added, it can be merged!

**Comment by ChielBruin** [comment]
Looks good now! Can be merged when travis finishes

**Pull request merged**